frontend/public/features.ts

```diff
@@ -120,11 +120,21 @@ const detectCanCreateProject = dispatch => coFetchJSON(projectRequestPath)
     }
   );
 
+const olmAPIGroupPath = `${k8sBasePath}/apis/${ClusterServiceVersionModel.apiGroup}/${ClusterServiceVersionModel.apiVersion}`;
+const detectOLMInstalled = dispatch => coFetchJSON(olmAPIGroupPath)
+  .then(
+    res => setFlag(dispatch, FLAGS.OPERATOR_LIFECYCLE_MANAGER, _.findIndex(res.resources, (({kind}) => kind === ClusterServiceVersionModel.kind)) > -1),
```

Comment:
Any reason to not just add this to the `CRDs` list above?

Reply:
Good call, updated.